tests/Agent/IntegrationTests/IntegrationTestHelpers/RemoteServiceFixtures/RemoteApplicationFixture.cs

```diff
@@ -86,7 +86,7 @@ public bool KeepWorkingDirectory
             set { RemoteApplication.KeepWorkingDirectory = value; }
         }
 
-        protected virtual int MaxTries => 3;
+        protected virtual int MaxTries => 1;
```

Comment:
I think a comment would be good here to help un-confuse the next developer to look at this code.

```suggestion
        // We think that the test retry loop may be masking real problems and/or actually causing them, so we've disabled it for now
        protected virtual int MaxTries => 1;
```